### PR TITLE
Added image to 1.10 img folder

### PR DESCRIPTION
**Pull request by anithamuthu79**
## Description
https://jira.mesosphere.com/browse/COPS-2104

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by adyatlov** [comment]
Hi, @anithamuthu79. Screenshots and paths to this information are different in different versions of DC/OS. The path and the screenshot you proposed are only valid for DC/OS version 1.11.

The change for the 1.11 looks OK, but I would move the screenshot to the 1.11 directory. For the other versions, I would say something like "You can also find the security mode in the DC/OS UI".

**Comment by adyatlov** [comment]
Hi @anithamuthu79. Thank you! "DC/OS UI" and "DC/OS Dashboard" are competing terms. Some people refer to the entire DC/OS UI as "DC/OS Dashboard" and some use this term only for the dashboard page (the first one if you log in to DC/OS as an admin).

I think in our case it's better to use the term "DC/OS UI" and get rid off the word "Dashboard".

File: pages/1.11/security/ent/index.md [hunk 1]
```diff
@@ -79,6 +79,10 @@ You can use either of the following methods to determine the security mode of an
 
 - [SSH](/1.11/administering-clusters/sshcluster/) into your master and view the contents of `/opt/mesosphere/etc/bootstrap-config.json`.
 
+- Use the DC/OS UI/Dashboard and go to:
+DC/OS UI -> Cluster -> Overview -> Scroll down to the bottom, to Bootstrap config: Security Mode strict.
+![SECURITY MODE strict](/1.10/img/discover-security-mode.png).
```
Comment (adyatlov) [inline pages/1.11/security/ent/index.md]: Thank you, Anitha,

As far as I understand, this is a screenshot of the DC/OS v1.11 and it's only shown on the page which corresponds to DC/OS v1.11. Could you please move it to the correspondent directory: `/1.10/img/`.

Comment (anithamuthu79) [reply pages/1.11/security/ent/index.md]: @adyatlov discover-security-mode.png is present in both 1.11 and 1.10 img folders. 
https://github.com/mesosphere/dcos-docs-site/pull/790

Comment (adyatlov) [reply pages/1.11/security/ent/index.md]: I'm sorry @anithamuthu79. I made a typo in my review. What I meant is that we need to move the `/1.10/img/discover-security-mode.png` file to `/1.11/img/discover-security-mode.png` and change the reference in the code from `SECURITY MODE strict.` to `SECURITY MODE strict.`

**Comment by ck4adventure** [comment]
Outdated, closing.

**Pull request closed without merging**